libevmasm/Assembly.cpp

```diff
@@ -798,20 +799,20 @@ std::map<u256, u256> const& Assembly::optimiseInternal(
 
 	// Run optimisation for sub-assemblies.
 	// TODO: verify and double-check this for EOF.
-	for (size_t subId = 0; subId < m_subs.size(); ++subId)
+	for (SubAssemblyID subID {0}; subID.value < m_subs.size(); ++subID.value)
```

Comment:
```suggestion
	for (SubAssemblyID subID{0}; subID.value < m_subs.size(); ++subID.value)
```

Also in `AssemblyItem::splitForeignPushTag()`.

Reply:
I really wish we could just settle on one style that has clang format support (or find a close enough approximation of the current style) and be done with it :) the amount of time and energy things like this eat up is unreasonable